lib/ci/container/ephemeral.rb

```diff
@@ -1,11 +1,13 @@
 require_relative '../container'
 
 module CI
+  class EphemeralContainerUnhandledState < StandardError; end
```

Comment:
I'd move that into the class

`CI::EphemeralContainer::UnhandledState` is more readable than `CI::EphemeralContainerUnhandledState`